lib/serverengine/daemon.rb

```diff
@@ -47,14 +47,23 @@ def initialize(server_module, worker_module, load_config_proc={}, &block)
       @chuser = @config[:chuser]
       @chgroup = @config[:chgroup]
       @chumask = @config[:chumask]
+
+      @pid = nil
+      @command_pipe = @config.fetch(:command_pipe, nil)
```

Comment:
`Daemon.run_server` calls `Daemon#server_main` instead of `Daemon#main`.
In this case, `@command_pipe` is used for receiving.
`@command_pipe` set in `Daemon#main` is used for sending (in other process).
But this is confusing.  I'll rename the sender pipe variable.